src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/controller/WelcomeControllerIntegrationTest.java

```diff
@@ -445,11 +446,11 @@ public void createOrgRoleMappingErrorWhenNoUsers() throws Exception {
                 .contentType(JSON_CONTENT_TYPE)
                 .headers(getHttpHeaders())
                 .content(mapper.writeValueAsBytes(request)))
-                .andExpect(status().is(404))
+                .andExpect(status().is(400))
                 .andReturn();
 
         String contentAsString = result.getResponse().getContentAsString();
-        assertTrue(contentAsString.contains("Resource not found Some of the user profiles couldn't be found"));
+        assertTrue(contentAsString.contains("The UserIds  is not available in the user's request"));
```

Comment:
Are we stopping our processing on this error? Processing should be continued if we receive atleast one user profile from CRD.

Reply:
At the dead point we are stopping the process.